.azure/applications/api/main.bicep

```diff
@@ -62,7 +62,7 @@ module databaseAccess '../../modules/postgreSql/AddAdministrationAccess.bicep' =
   params: {
     tenantId: appIdentity.outputs.tenantId
     principalId: appIdentity.outputs.principalId
-    appName: appIdentity.name
```

Comment:
To avoid duplicating the identity name definition, it would be better to use appIdentity.outputs like above. But need to add name as an output variable to modules/identity/create.bicep first.